### ISSUE TITLE
IntelliJ Rust falsely reports that the error_chain module is private but it compiles and works

### ISSUE THREAD

**Issue by lockieluke** [intellij-rust/intellij-rust#7124]
## Environment

* **IntelliJ Rust plugin version:** 0.3.145.3797-211
* **Rust toolchain version:** null
* **IDE name and version:** IntelliJ IDEA 2021.1 Community Edition (IC-211.6693.111)
* **Operating system:** macOS 10.14.6
* **Macro expansion engine:** new
* **Name resolution engine:** old

## Problem description
So as the title says, the error_chain module is reported that it is private, but it's not true, it can be compiled and most importantly it works.

## Steps to reproduce
Import error chain like this:
```rust
use error_chain::error_chain;
```

![image](https://user-images.githubusercontent.com/25424409/115024985-5ed21580-9ef3-11eb-97e5-ccc9be2e4efa.png)

**Comment by lockieluke** [comment]
I changed to the experimental name resolution engine and it didn't fix it too.
